TShockAPI/Bouncer.cs

```diff
@@ -2496,5 +2511,45 @@ internal static int GetMaxPlaceStyle(int tileID)
 			{ ProjectileID.GoldShortswordStab, ItemID.GoldShortsword },
 			{ ProjectileID.PlatinumShortswordStab, ItemID.PlatinumShortsword }
 		};
+
+		private Dictionary<short, short> Projectile_MinValuesAI = new Dictionary<short, short> {
+			{ 611, -1 },
+
+			{ 950, 0 }
+		};
+		private Dictionary<short, short> Projectile_MaxValuesAI = new Dictionary<short, short> {
+			{ 611, 1 },
+
+			{ 950, 0 }
+		};
+
+		private Dictionary<short, short> Projectile_MinValuesAI2 = new Dictionary<short, short> {
+			{ 405, 0 },
+			{ 410, 0 },
```

Comment:
```suggestion
			{ 410, 0.6f },
```
Source: `Projectile.AI()`, `this.ai[1] = (float)Main.rand.Next(60, 121) / 100f;`